scripts/Dockerfile.devel

```diff
@@ -1,10 +1,17 @@
 FROM ceph/ceph:v15
 
-ENV GOPATH=/go \
-	GO111MODULE=on
+
+RUN \
+ mkdir -p /go && \
```

Comment:
can you handle the golang version via arguments for dockerfile?

Reply:
Is it fine to do it in follow up PR ? @Madhu-1